README.md

```diff
@@ -255,6 +255,30 @@ A `TockTheme` can be used as a value of a `ThemeProvider` of [`emotion-theming`]
 | `timeoutBetweenMessage`                  | `number?`                             | Timeout between message                                          |
 | `widgets`                                | `any?`                                | Custom display component                                         |
 | `disableSse`                             | `boolean?`                            | Disable SSE (not even trying)                                    |
+| `sessionStorage`                         | `boolean?`                            | Enable history session storage                                   |
+
+#### Session storage
+
+The optional `sessionStorage` makes it possible to provide a history session of messages.
+This history loads at the creation of the chat and is stored in the locale storage of the browser.
+
+The `sessionStorage` parameter is a boolean, by default set to false.
```

Comment:
Hello @correi-f very nice feature, I just think the `sessionStorage` option might be renamed to `localStorage`, which I find less ambiguous (since the conversation actually stores to the local storage).

Reply:
I agree with you